kolibri/content/utils/annotation.py

```diff
@@ -1,38 +1,163 @@
-from kolibri.utils.time import local_now
+import logging as logger
+import math
+import os
+
 from django.conf import settings
+from kolibri.content.apps import KolibriContentConfig
+from kolibri.content.models import ChannelMetadata, ContentNode, File, LocalFile
+from le_utils.constants import content_kinds
+from sqlalchemy import and_, select
 
-from ..content_db_router import using_content_database
-from ..models import ChannelMetadata, ChannelMetadataCache
 from .channels import get_channel_ids_for_content_database_dir
+from .paths import get_content_file_name, get_content_storage_file_path
+from .sqlalchemybridge import Bridge
+
+logging = logger.getLogger(__name__)
 
+CONTENT_APP_NAME = KolibriContentConfig.label
 
 def update_channel_metadata_cache():
     """
-    After a channel is imported, or when the devserver is started,
+    If we are potentially moving from a version of Kolibri that did not import its content data,
     scan through the settings.CONTENT_DATABASE_DIR folder for all channel content databases,
     and pull the data from each database's ChannelMetadata object to update the ChannelMetadataCache
```

Comment:
That's one scenario where this would get called, but I can also imagine it would be useful in cases where somebody copies in a new content folder (or moves a DB from one device to another), and we need to "resync".